project2-beat-that/script.js

```diff
@@ -1,4 +1,299 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+var rollDice = function () {
+
+  var randomDecimal = Math.random() * 6;
+
+  var randomInteger = Math.floor(randomDecimal);
+
+  var diceNumber = randomInteger + 1;
+
+  return diceNumber;
+};
+
+//Game states 
+var INTRODUCTION = 'INTRODUCTION'
+var ROLL_DICE = 'ROLL DICE';
+var CHOOSE_DICE_ORDER = 'CHOOSE DICE ORDER';
+var CHANGE_TO_PLAYER_2 = 'CHANGE TO PLAYER 2';
+var CHOOSE_DICE_ORDER_PLAYER_2 = 'CHOOSE DICE ORDER PLAYER 2';
+var COMPARE_PLAYERS = `COMPARE PLAYERS`;
+var CONTINUE_GAME = `CONTINUE GAME`; 
+var LOWEST_COMBINE_NUMBDER = `LOWEST COMBINE NUMBDER`;
+var AUTO_COMBINE_NUMBER = `AUTO COMBINE NUMBER`
+var SCORE = `SCORE`
```

Comment:
nice use of SCREAM_CASE to denote these constants

consider using the const keyword also, rather than var